Gemfile.lock

```diff
@@ -1,31 +1,31 @@
 PATH
   remote: .
   specs:
-    moderate_parameters (0.4.1)
-      actionpack (>= 4.2, < 6.2)
-      activemodel (>= 4.2, < 6.2)
-      activesupport (>= 4.2, < 6.2)
-      railties (>= 4.2, < 6.2)
+    moderate_parameters (0.4.2)
+      actionpack (>= 4.2, < 7.1)
+      activemodel (>= 4.2, < 7.1)
+      activesupport (>= 4.2, < 7.1)
+      railties (>= 4.2, < 7.1)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.1.4.1)
-      actionview (= 6.1.4.1)
-      activesupport (= 6.1.4.1)
+    actionpack (6.1.5)
```

Comment:
Note: it's probably worth checking whether these versions make sense or not. This gem is fine when loaded into my "upgrading to rails 7.0" project, but not sure (a) what these versions should be set to, and (b) whether I ran `bundle` properly to generate this `Gemfile.lock` for future development.